workflows/prognostic_c48_run/runtime/segmented_run/prepare_config.py

```diff
@@ -197,26 +196,6 @@ def _create_arg_parser() -> argparse.ArgumentParser:
         help="Path to a config update YAML file specifying the changes from the base"
         "fv3config (e.g. diag_table, runtime, ...) for the prognostic run.",
     )
-    parser.add_argument(
-        "--model_url",
-        type=str,
-        default=None,
-        action="append",
-        help=(
-            "Remote url to a trained ML model. If a model is omitted (and not "
-            "specified in `user_config`'s `scikit-learn` `model` field either), then "
-            "no ML updating will be done. Also, if an ML model is provided, no "
-            "nudging will be done. Can be provided multiple times, "
-            "ex. --model_url model1 --model_url model2. If multiple urls are given, "
-            "they will be combined into a single model at runtime, providing the "
-            "outputs are nonoverlapping."
-        ),
-    )
-    parser.add_argument(
```

Comment:
If this info is not in the docs, can you move this info there so it is still somewhere discoverable?

Reply:
The option is documented in the `MachineLearningConfig` dataclass, which is included in the API page on the docs.